ui/cypress/e2e/onboarding.test.ts

```diff
@@ -122,74 +130,88 @@ describe('Onboarding', () => {
       .should('be.enabled')
     cy.get('button:contains("Continue")').click()
 
-    //wait for new page to load
-    cy.location('pathname', {timeout: defTimeOut}).should(
-      'include',
-      'onboarding/2'
-    )
+    cy.wait('@orgSetup')
 
-    cy.getByTestID('notification-success').should($msg => {
-      expect($msg).to.contain('Initial user details have been successfully set')
-    })
+    //used in assertion below
 
-    cy.get('div[data-testid=notification-success] span.icon.checkmark').should(
-      'be.visible'
-    )
-    cy.get(
-      'div[data-testid=notification-success] button.notification-close'
-    ).should('be.visible')
+    cy.get('@orgSetup').then((xhr) => {
 
-    //check navbar
-    cy.get('div.wizard--progress-title.current').should($div => {
-      expect($div).to.contain('Complete')
-    })
-    cy.get('div.wizard--progress-title.checkmark').should($div => {
-      expect($div).to.contain('Welcome')
-      expect($div).to.contain('Initial User Setup')
-    })
+      let orgId: string = xhr.responseBody.org.id
 
-    cy.get('button.button-success')
-      .contains('Quick Start')
-      .should('be.visible')
-    cy.get('button.button-success')
-      .contains('Advanced')
-      .should('be.visible')
-    cy.get('button.button-success')
-      .contains('Configure Later')
-      .should('be.visible')
-
-    //advance to Quick Start
-    cy.get('button.button-success')
-      .contains('Quick Start')
-      .click()
-
-    cy.location('pathname', {timeout: defTimeOut}).should('include', '/me')
-
-    cy.getByTestID('notification-success').should($msg => {
-      expect($msg).to.contain(
-        'The InfluxDB Scraper has been configured for http://localhost:9999/metrics'
+      //wait for new page to load
+      cy.location('pathname').should(
+        'include',
+        'onboarding/2'
       )
-    })
 
-    cy.get('div[data-testid=notification-success] span.icon.checkmark').should(
-      'be.visible'
-    )
+      cy.getByTestID('notification-success').should($msg => {
+        expect($msg).to.contain('Initial user details have been successfully set')
+      })
 
-    cy.get(
-      'div[data-testid=notification-success] button.notification-close'
-    ).should('be.visible')
+      cy.get('div[data-testid=notification-success] span.icon.checkmark').should(
+        'be.visible'
+      )
+      cy.get(
+        'div[data-testid=notification-success] button.notification-close'
+      ).should('be.visible')
+
+      //check navbar
+      cy.get('div.wizard--progress-title.current').should($div => {
+        expect($div).to.contain('Complete')
+      })
+      cy.get('div.wizard--progress-title.checkmark').should($div => {
+        expect($div).to.contain('Welcome')
+        expect($div).to.contain('Initial User Setup')
```

Comment:
we dont need to test this exact text.  this is highly likely to change and cause this test to be overly brittle.

Reply:
From the Cypress best practices,  using the rule of thumb question "If this text changed would I want the test to fail?"  I would assume that if the step labels changed we would want this to fail.  If you still disagree than these can be changed or removed.